app/templates/components/explore/side-bar.hbs

```diff
@@ -1,3 +1,11 @@
+<LeafletMap @lat={{this.lat}} @lng={{this.lng}} @zoom={{15}} as |layers|>
+  <layers.tile @url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" @attribution="&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors" />
+  <layers.marker @location={{array this.lat this.lng}} as |marker|>
+    <marker.popup>
+      {{this.location}}
+    </marker.popup>
+  </layers.marker>
+</LeafletMap>
```

Comment:
Duplication from EventMap. Extract to remove duplicate code

Reply:
> It's just OSM, mapbox will be added later

ok